client/webpack.config.js

```diff
@@ -355,6 +376,18 @@ const webpackConfig = {
 		shouldProfile && new webpack.ProgressPlugin( { profile: true } ),
 
 		shouldUsePersistentCache && shouldUseReadonlyCache && new ReadOnlyCachePlugin(),
+
+		// NOTE: Must be last.
+		shouldCreateSentryRelease &&
+			new SentryCliPlugin( {
+				org: 'a8c',
+				project: 'calypso',
+				authToken: process.env.SENTRY_AUTH_TOKEN,
+				release: process.env.COMMIT_SHA,
+				include: filePaths.path,
+				urlPrefix: `~${ filePaths.publicPath }`,
+				configFile: 'sentry.properties',
```

Comment:
What's this for?

Reply:
Oh, nice catch. I didn't mean to add that; I think it must be left over from a copy/paste.